meta-rdk-mtk-bpir4/recipes-ccsp/ccsp/ccsp-common-library.bbappend

```diff
@@ -46,6 +46,14 @@ do_install_append_class-target() {
 
    #SNMP SUPPORT
    sed -i "/tcp\:192.168.254.253\:705/a  ExecStart=\/usr\/bin\/snmp_subagent \&" ${D}${systemd_unitdir}/system/snmpSubAgent.service
+  
+   if ${@bb.utils.contains('DISTRO_FEATURES', 'partner_default_ext', 'true', 'false', d)}; then
```

Comment:
do we have similar distro in rpi

Reply:
yes chandra..
Please see the below link for your reference,
https://code.rdkcentral.com/r/plugins/gitiles/rdk/components/generic/rdk-oe/meta-cmf-raspberrypi/+/refs/heads/rdk-next/meta-rdk-broadband/recipes-ccsp/ccsp/ccsp-common-library.bbappend#185